context/app/api/vitessce_confs/test_confs.py

```diff
@@ -16,32 +16,66 @@
 FIXTURES_INPUT_DIR = "fixtures/input_entity"
 FIXTURES_EXPECTED_OUTPUT_DIR = "fixtures/output_expected"
 
+BASE_NAME_FOR_SPRM = "BASE_NAME"
+
 AssayConfClasses = {
     "codex": CytokitSPRMConf,
     "rna": RNASeqConf,
     "atac": ATACSeqConf,
     "ims": IMSConf,
     "image_pyramid": ImagePyramidViewConf,
     "seqfish": SeqFISHViewConf,
+    "sprm": SPRMViewConf,
 }
 
 
 def test_assays():
     parent_dir = Path(__file__).parent
-    for entity_file in (parent_dir / FIXTURES_INPUT_DIR).glob("*_entity.json"):
+    for entity_file in (parent_dir / FIXTURES_INPUT_DIR).glob(
+        "[!malformed]*_entity.json"
```

Comment:
No: This doesn't exclude files beginning with `malformed`, it excludes files beginning with `m` or `a` or `l` ...